beacon-chain/slasher/chunks_test.go

```diff
@@ -90,7 +90,7 @@ func TestMinSpanChunksSlice_CheckSlashable(t *testing.T) {
 	validatorIdx := types.ValidatorIndex(1)
 	source := types.Epoch(1)
 	target := types.Epoch(2)
-	att := createAttestation(source, target)
+	att := createAttestationWrapper(source, target, nil, nil)
```

Comment:
small comment to explain what the nils are could be helpful, something like (source, target, nil /* indices */, nil /* signing root */)